tests/tasks/setup/kit-cluster/default.yaml

```diff
@@ -81,25 +84,62 @@ spec:
               - name: scheduler
                 args:
                   - --kube-api-qps=$(params.scheduler-qps)
+        etcd:
+          spec:
+            nodeSelector: 
+              node.kubernetes.io/instance-type: $(params.cp-instance-type)
```

Comment:
Should this be `etcd-instance-type`? We will need to have a new param for this.

Reply:
Currently we have same param for both. Atleast in my testing, I only use same instance type for CPIs and etcds. But i see why we might need it for other testing purposes. I will add that in next PR.